10.9-libcxx/stable/main/finkinfo/graphics/netpbm11-shlibs.info

```diff
@@ -11,27 +11,38 @@ BuildDepends: <<
 	libiconv-dev,
 	libjasper.1,
 	libjbig,
-	libjpeg8-turbo,
+	(%m != arm64) libjpeg8-turbo-shlibs,
+	(%m = arm64) libjpeg9-shlibs,
+	libjpeg8-turbo | libjpeg9,
```

Comment:
Why the BDeps on the -shlibs packages? Shouldn't the %m conditionals be on the header packages here rather than using the `|` for dev pkg alternatives?

Reply:
Yes; typo (or rather leftovers from copy-and-paste)!